src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -41,7 +41,7 @@ export interface IncorporationFilingIF {
       phone: string
       extension?: number
     }
-    parties: OrgPersonIF[]
+    parties: PartyIF[]
```

Comment:
OrgPersonIF is close enough to PartyIF that IA filings and dissolution filings previously worked, but the [incorporation application](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/incorporation_application.json) and [dissolution](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/dissolution.json) schemas do, in fact, specify the Party schema/interface.

TLDR: this change doesn't break IAs or dissolutions and is now correct.